src/xtgeo/grid3d/_grid_roxapi.py

```diff
@@ -35,18 +34,18 @@
 # ======================================================================================
 
 
-def import_grid_roxapi(
+def load_grid_from_rms(
```

Comment:
Renamed functions, thinking "load" is more human readable than "import" in this context

Simialr for "export" which is rename "save"